src/icon/supportIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { color } from '../_utils/branding'
+import { BaseIcon, BaseIconDefaultProps, Icon } from '../_utils/icon'
+
+export const SupportIcon = (props: Icon) => (
```

Comment:
Icon names should be representative of its drawing than its function:
eg. this is a `headset` and can be used to represent other things than a "support"